.github/workflows/build-vcpkg.yml

```diff
@@ -123,6 +111,77 @@ jobs:
           path: ${{ runner.workspace }}/build/tfs
         if: "! contains( matrix.os, 'windows')"
 
+      - name: Prepare datapack contents
+        run: find . -maxdepth 1 ! -name data ! -name config.lua.dist ! -name key.pem ! -name LICENSE ! -name README.md ! -name schema.sql -exec rm -r {} \;
+        shell: bash
+
+      - name: Upload datapack contents
+        uses: actions/upload-artifact@v2
+        with:
+          name: tfs-${{ matrix.name }}-${{ matrix.buildtype }}-luajit=${{ matrix.luajit }}-${{ github.sha }}
+          path: ${{ github.workspace }}
+
+  windows:
+    name: ${{ matrix.os }}-${{ matrix.cxx }}-${{ matrix.buildtype }}-luajit=${{ matrix.luajit }}
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 8
+      matrix:
+        name: [windows-msvc]
+        buildtype: [Debug, Release]
+        luajit: [on]
```

Comment:
You can drop this from the matrix and from the job name, unless it is used for something else